go.mod

```diff
@@ -37,11 +37,11 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20191217072959-393e6c0fd4b7
 	github.com/pingcap/log v0.0.0-20191012051959-b742a5d432e9
-	github.com/pingcap/parser v0.0.0-20191223023445-b93660cf3e4e
+	github.com/pingcap/parser v0.0.0-20191223211552-4cf10469de1d
```

Comment:
Did you run 

```
go get github.com/pingcap/parser@master
```

Why many other irrelevant packages are changed?

Reply:
yes, i added `GlobalDBVisible` into parser repo and run go get.

Reply:
github.com/pingcap/tipb is updated, that's weird